app/helpers/react_components/mentoring/discussion.rb

```diff
@@ -44,7 +45,9 @@ def iterations
             idx: iteration.idx,
             num_comments: ccs.sum(&:second),
             unread: ccs.reject { |(_, seen), _| seen }.present?,
+            created_at: iteration.created_at.iso8601,
```

Comment:
Where is this used in the UI? Just wondering if it should be created_at, or updated_at

Reply:
This is to show when the iteration was submitted.